### PR TITLE
move from apache to nginx in docker

### PR DESCRIPTION
**Pull request by tdieterich-drip**
Addresses [ECI-208](https://dripcom.atlassian.net/browse/ECI-208)
moving to nginx allows us to use XDebug in remote-mode while not impacting peformance  (... _too much_ ...)

### CODE REVIEW

File: devtools_m2/cypress/support/docker_helpers.js [hunk 1]
```diff
@@ -5,5 +5,5 @@ Cypress.Commands.add("flushCaches", () => {
 
 beforeEach(function() {
   cy.log('resetting docker for test')
-  cy.exec('DRIP_COMPOSE_ENV=test ./reset.sh')
+  cy.exec('DRIP_COMPOSE_ENV=test ./reset.sh', {timeout: 120000})
```
Comment (tdieterich-drip) [inline devtools_m2/cypress/support/docker_helpers.js]: While this is typically executing within 20 seconds on my machine, sometimes it takes quite awhile to run, and I have no idea why.  However, this seems to be long enough to let it complete.

File: devtools_m2/docker-compose.base.yml [hunk 2]
```diff
@@ -14,6 +14,7 @@ services:
       - type: bind
         source: './..'
         target: /var/www/html/magento/app/code/Drip/Connect
+        consistency: cached
```
Comment (tdieterich-drip) [inline devtools_m2/docker-compose.base.yml]: This makes the host (your local machine, not the docker container) the authority on what should appear in `.../Drip/Connect`.  It reduces over head in the docker container, which [theoretically] helps keep the PHP+Xdebug environment performant.

File: devtools_m2/cypress/support/commands.js [hunk 3]
```diff
@@ -28,7 +28,7 @@ afterEach(function() {
   // https://github.com/cypress-io/cypress/issues/216
   // https://github.com/cypress-io/cypress/issues/686
   cy.log('Reset web interface to avoid bleed-over')
-  cy.visit('/lib/web/blank.html', { failOnStatusCode: false })
+  cy.visit('/lib/web/blank.html', { failOnStatusCode: false  })
```
Comment (will-in-wi) [inline devtools_m2/cypress/support/commands.js]: Nit: Could you fix this whitespace?

File: devtools_m2/cypress/support/docker_helpers.js (hunk 1 above)
Comment (will-in-wi) [reply devtools_m2/cypress/support/docker_helpers.js]: Might be worth annotating the reset file to have better logs…

Comment (tdieterich-drip) [reply devtools_m2/cypress/support/docker_helpers.js]: The reset file itself is pretty simple (i.e. there's not a lot to annotate).  The long execution is in the two calls to Magento, not the DB.

File: devtools_m2/cypress/support/commands.js (hunk 3 above)
Comment (tdieterich-drip) [reply devtools_m2/cypress/support/commands.js]: please tell me you have some sort of automation around catching this...

Comment (will-in-wi) [reply devtools_m2/cypress/support/commands.js]: Not yet. But we could pretty easily throw eslint around these files.

**Pull request merged**